fuel-relayer/src/pending_blocks.rs

```diff
@@ -617,11 +641,27 @@ mod tests {
         let b2 = rng.gen();
 
         let mut blocks = block_commit(1u64.into());
-        let mut db = Box::new(DummyDb::filled());
+        let mut db = Box::new(MockDb::default());
         blocks.handle_block_commit(b1, 2u64.into(), 2, IsReverted::False);
         blocks.handle_block_commit(b2, 3u64.into(), 3, IsReverted::False);
         blocks.handle_block_commit(b2, 3u64.into(), 3, IsReverted::True);
 
+        db.tap_sealed_blocks_mut(|sealed_blocks| {
```

Comment:
would it make sense to have a helper test function that takes N amount of blocks and creates them instead of repating this process in multiple tests?

Reply:
Personally, I prefer having the test setup localized to the test itself, even if it means repeated code. Extra helper functions make setup less localized, and may obfuscate test setup (even if it's just by a little bit). The spirit of this PR is to be explicit in the test setup and to increase code locality (by removing the DummyDb and its implicit setup) and I think that such an approach would be counter to that. 
I understand the suggestion, but personally would like to avoid it if I can!